app/Resources/views/full/contact_form.html.twig

```diff
@@ -73,22 +73,24 @@
             </div>
         </div>
     </section>
+{% endblock %}
 
+{% block javascripts %}
     <script src="http://maps.googleapis.com/maps/api/js"></script>
-    <script>
+    <script type="text/javascript">
```

Comment:
This is HTML5, we don't need `type="text/javascript"` anymore (it is used by default).